pvlib/location.py

```diff
@@ -44,8 +45,10 @@ class Location:
         pytz.timezone objects will be converted to strings.
         ints and floats must be in hours from UTC.
 
-    altitude : float, default 0.
+    altitude : None or float, default None
         Altitude from sea level in meters.
+        If None, the altitude will be fetched from
+        :py:func:`pvlib.location.lookup_altitude`.
```

Comment:
We might want to add a note that `altitude` is set to 0 if there is no data at the location.